README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
+asdf
```

Comment:
The addition of 'asdf' is not appropriate content for a README file. README files should contain clear, informative content about the project. Please replace this with relevant information or remove it if not needed.